src/components/home/sections/sections.scss

```diff
@@ -8,14 +8,30 @@
   flex-direction: column;
   align-items: center;
   height: 85vh;
-
+  @media only screen and (max-width: 375px){
```

Comment:
This is a comment for the whole file. All your `@media` breakpoints are scattered and unorganized which makes it hard to remember what's been covered and what hasn't. I'm going to send you what I have and we can discuss it.